tests/login.spec.js

```diff
@@ -0,0 +1,28 @@
+const { test, expect } = require('@playwright/test');
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+});
+
+test('login', async ({ page }) => {
+    await page.goto('https://www.saucedemo.com/');
+
+    //login
+    await page.getByPlaceholder('Username').click();
+    await page.getByPlaceholder('Username').fill('standard_user');
+    await page.getByPlaceholder('Password').click();
+    await page.getByPlaceholder('Password').fill('secret_sauce');
+    await page.locator('#login-button').click();
+
+    //verify visibility
+    await expect(page.getByText('Sauce Labs Backpack')).toBeVisible();
+    await page.locator('[data-test="shopping-cart-link"]').click();
```

Comment:
'data-test' attribute can be set in the playwright config file like:
```
use: {
    testIdAttribute: 'data-test',
...
}
```
And then you can use it like 
```await page.getByTestId('shopping-cart-link').click();```
Same for login button

But I don't see this action in the unit task, so maybe you need to read it one more time 
You need to: 
- Perform login
- Check that the "Products" title is displayed
- Check that shopping cart is displayed
- And that there are more than one product on the page

No need to go to the cart